waspc/src/Wasp/Util/StrongPath.hs

```diff
@@ -33,3 +37,9 @@ splitAbsExtension path =
 splitRelExtension :: MonadThrow m => SP.Path' (SP.Rel b) (SP.File a) -> m (SP.Path' (SP.Rel b) (SP.File c), String)
 splitRelExtension path =
   first SP.fromPathRelFile <$> P.splitExtension (SP.toPathRelFile path)
+
+findAllFilesWithSuffix :: String -> [SP.Path p r (SP.File f)] -> [SP.Path p r (SP.File f)]
+findAllFilesWithSuffix extension = filter ((extension `isSuffixOf`) . SP.toFilePath)
+
+getFilename :: SP.Path' (SP.Rel b) (SP.File a) -> String
+getFilename = takeFileName . SP.toFilePath
```

Comment:
I don't think we need this function. `fromRelFile` from `StrongPath` does the same thing. We can just use that in all places we're using this.

Or am I missing something?

Reply:
I've removed it.